internal/indexer/definitions/ptn.yaml

```diff
@@ -5,19 +5,19 @@ identifier: ptn
 description: PTN is a ratioless movie tracker.
 language: en-us
 urls:
-  - http://piratethenet.org/
   - https://piratethenet.org/
+  - http://piratethenet.org/
 privacy: private
 protocol: torrent
 supports:
   - irc
   - rss
 source: custom
 settings:
-  - name: torrent_pass
+  - name: passkey
     type: secret
     required: true
-    label: Torrent pass
+    label: Passkey
```

Comment:
Lets keep name as torrent_pass to not break existing users but keep the new label. Set it in the torrentUrl as well